M2/Macaulay2/packages/Hadamard.m2

```diff
@@ -0,0 +1,441 @@
+newPackage(
+    "Hadamard",
+    Version => "1.0",
```

Comment:
If this package is preliminary (it seems so), then the version number can be less than 1.

Reply:
I changed it to V 0.1